views/class.html

```diff
@@ -36,6 +36,13 @@
           <br>{{project.due}}
         </li>
         <li class="collection-item" ng-show="!overdue_projects.length"><i>No overdue projects!</i></li>
+        <li class="collection-item" ng-show="class.owner == current_user">
+          <a href="#/classes/{{class.url_name}}/projects/create/">
+            <button class="waves-effect waves-light btn lighten-1 white-text">
```

Comment:
I'd recommend adding the `blue` class so that the button looks consistent with the rest of the site.